batching.go

```diff
@@ -14,30 +14,27 @@ type batch struct {
 }
 
 // Batch creates a set of deferred updates to the database.
-func (d *Datastore) Batch() (ds.Batch, error) {
-	return &batch{ds: d, batch: &pgx.Batch{}}, nil
+func (d *Datastore) Batch(ctx context.Context) (ds.Batch, error) {
+	b := &pgx.Batch{}
+	b.Queue("BEGIN")
```

Comment:
This change seems to be changing the semantics of method calls? It looks like previously you could do:
Batch()
Put()
Get()

And now you can't. Maybe it deserves some comment documentation?

Reply:
This is because the go-datastore interface changed to require contexts for those methods. I don't think it makes sense to add a comment for that, it is implied by the fact that this implements the datastore interface, and it has changed.

Also consider that this is a new module and so has no consumers, so this is not a breaking change either.